<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyPrinter</class>
 <widget class="QWidget" name="ModifyPrinter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>470</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="buddy">
        <cstring>nameLE</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KLineEdit" name="nameLE">
       <property name="toolTip">
        <string>May contain any printable characters except &quot;/&quot;, &quot;#&quot;, and space</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="buddy">
        <cstring>descriptionLE</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KLineEdit" name="descriptionLE">
       <property name="toolTip">
        <string>Human-readable description such as &quot;HP LaserJet with Duplexer&quot;</string>
       </property>
       <property name="AttributeName" stdset="0">
        <string notr="true">printer-info</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Location:</string>
       </property>
       <property name="buddy">
        <cstring>locationLE</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KLineEdit" name="locationLE">
       <property name="toolTip">
        <string>Human-readable location such as &quot;Lab 1&quot;</string>
       </property>
       <property name="AttributeName" stdset="0">
        <string notr="true">printer-location</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="connectionL">
       <property name="text">
        <string>Connection:</string>
       </property>
       <property name="buddy">
        <cstring>connectionLE</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="KLineEdit" name="connectionLE">
       <property name="whatsThis">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Examples:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;http://hostname:631/ipp/&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;http://hostname:631/ipp/port1&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;ipp://hostname/ipp/&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;ipp://hostname/ipp/port1&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;lpd://hostname/queue&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;socket://hostname&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;socket://hostname:9100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="AttributeName" stdset="0">
        <string notr="true">device-uri</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="membersL">
       <property name="text">
        <string>Members</string>
       </property>
       <property name="buddy">
        <cstring>membersLV</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="ClassListWidget" name="membersLV"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="driverL">
       <property name="text">
        <string>Driver:</string>
       </property>
       <property name="buddy">
        <cstring>makeCB</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="KComboBox" name="makeCB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ClassListWidget</class>
   <extends>QListView</extends>
   <header>ClassListWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameLE</tabstop>
  <tabstop>descriptionLE</tabstop>
  <tabstop>locationLE</tabstop>
  <tabstop>connectionLE</tabstop>
  <tabstop>membersLV</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
